### PR TITLE
Export in global scope

### PR DESCRIPTION
**Pull request by mkazlauskas**
> This pull request is for Electron, where it won't export to global scope when imported as <script>

### Background & Context

Also removes deprecated version field in bower.json

### CODE REVIEW

**Comment by cure53** [comment]
I think the version numbers are highly outdated :)

@fhemberger What do you think about the other change? I lack expertise here to judge it properly.

File: src/purify.js [hunk 1]
```diff
@@ -7,9 +7,8 @@
         define(function(){ return factory(root); });
     } else if (typeof module !== 'undefined') {
         module.exports = factory(root);
-    } else {
-        root.DOMPurify = factory(root);
     }
+    root.DOMPurify = factory(root);
```
Comment (fhemberger) [inline src/purify.js]: Tests are failing:

```
/home/travis/build/cure53/DOMPurify/src/purify.js:11
    root.DOMPurify = factory(root);
                   ^
TypeError: Cannot set property 'DOMPurify' of null
    at /home/travis/build/cure53/DOMPurify/src/purify.js:11:20
    at Object.<anonymous> (/home/travis/build/cure53/DOMPurify/src/purify.js:12:2)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/cure53/DOMPurify/test/jsdom-node.js:7:17)
    at Module._compile (module.js:413:34)
```

Comment (fhemberger) [inline src/purify.js]: I don't know what the issue is with Electron. Can you please elaborate a bit more?

**Comment by cure53** [comment]
Any info or news here? @mkazlauskas

**Comment by mkazlauskas** [comment]
I'm not sure how your tests fail. What my change intended to do is export the library to global scope (window) in Electron environment in the same way as in regular browser environment. Since Electron has both `require` and `window` it kinda makes sense to export to both. Also, in my specific use case I must use it from global environment and can't use require (doing HTML Imports to load it).

**Comment by cure53** [comment]
cc @fhemberger @neilj

**Comment by cure53** [comment]
Given the massive changes in #206, does this PR still make sense?

File: src/purify.js (hunk 1 above)
Comment (tdeekens) [inline src/purify.js]: We should not interact with the `factory` of the surrounding UMD build in the first place. This has changed after merging #206 where we export a function `createDOMPurify` function you can use to wrap the library onto another global. You can see an example in our test setup for jsdom [here](https://github.com/cure53/DOMPurify/blob/master/test/jsdom-node.js#L6).

**Comment by cure53** [comment]
Closing this for now, feel free to re-open if this still makes sense.

**Pull request closed without merging**